Q3 small signin updates (#16383)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/signin-confirmcompromised.md

---
title: "signIn: confirmCompromised"

> description: "Allows you to mark Azure AD log events as risky for Azure AD Identity Protection."

Reviewer: I think not explicitly mentioning sign ins infers this action is also for other log types.
```suggestion
description: "Allows you to mark Azure AD sign in events as risky for Azure AD Identity Protection."
```

author: "besiler"
ms.localizationpriority: medium
ms.prod: "identity-and-access-reports"
doc_type: apiPageType
---

# signIn: confirmCompromised
Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> This API allows admins to mark an event in the Azure AD logs as risky. Events marked as risky by an admin are immediately flagged as high risk in Azure AD Identity Protection, overriding previous risk states. Admins can confirm that events flagged as risky by Azure AD Identity Protection are in fact risky or can mark unflagged events as risky. To mark a sign-in as risky, include he requestID of the sign-in in the request body. If successful, this method returns a 204 No Content response code. It does not return anything in the response body.

Reviewer: Suggestion to remove the last 2 sentences. Those will be catered for in the Request body and Response sections below.
```suggestion
Allows admins to mark an event in the Azure AD sign in logs as risky. Events marked as risky by an admin are immediately flagged as high risk in Azure AD Identity Protection, overriding previous risk states. Admins can confirm that events flagged as risky by Azure AD Identity Protection are in fact risky or they can mark unflagged events as risky. 
```

## Permissions
One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|IdentityRiskyUser.ReadWrite.All|
|Delegated (personal Microsoft account)|Not supported.|
|Application|IdentityRiskyUser.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
POST /auditLogs/signIns/confirmCompromised
```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|
|Content-Type|application/json. Required.|

## Request body
In the request body, supply JSON representation of the parameters.

The following table shows the parameters that can be used with this action.

|Parameter|Type|Description|
|:---|:---|:---|

> |requestIds|String collection|The requestIds of the events that should be marked risky for Azure AD Identity Protection.|

Reviewer: I believe you're referring to the value in the **id** property?
```suggestion
|requestIds|String collection|The IDs of the sign in events that should be marked risky for Azure AD Identity Protection.|
```

Author: Yep, that's correct

## Response

If successful, this action returns a `204 No Content` response code.

## Examples

### Request
<!-- {
  "blockType": "request",
  "name": "signinthis.confirmcompromised"
}
-->
``` http
POST https://graph.microsoft.com/beta/auditLogs/signIns/confirmCompromised
Content-Type: application/json

> Content-length: 42

Reviewer: Let's remove this.
```suggestion
```

{
  "requestIds": [
    "f01c6af6-6683-4a37-a945-0a925501eede",
    "42bf60ac-d0cb-4206-aa5c-101884298f55",
    "f09c8f14-8d8e-42cf-8a7e-732b0594e79b"
  ]
}
```

### Response

> >**Note:** The response object shown here might be shortened for readability.

Reviewer: ```suggestion
```
